crates/networking/p2p/snap.rs

```diff
@@ -72,18 +70,16 @@ pub fn process_storage_ranges_request(
         // Generate proofs only if the response doesn't contain the full storage range for the account
         // Aka if the starting hash is not zero or if the response was capped due to byte limit
         if !request.starting_hash.is_zero() || res_capped && !account_slots.is_empty() {
-            proof.extend(
+            proof.extend(proof_to_encodable(
                 store
                     .get_storage_range_proof(
                         request.root_hash,
                         hashed_address,
                         request.starting_hash,
                         account_slots.last().map(|acc| acc.hash),
                     )?
-                    .unwrap_or_default()
-                    .iter()
-                    .map(|bytes| Bytes::copy_from_slice(bytes)),
-            );
+                    .unwrap_or_default(),
```

Comment:
Nit: Can we use the actual default value here instead of _or_default?
I think it's a bit more declarative this way.

Reply:
I don't see how `unwrap_or(ver![])` is any more clear (it looks like something clippy would complain about)

Reply:
Nevermind then